frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/details-card.tsx

```diff
@@ -57,10 +57,13 @@ const DetailsCard: React.FC<DashboardItemProps> = ({
   const infrastructure = _.get(resources, 'infrastructure');
   const infrastructureLoaded = _.get(infrastructure, 'loaded', false);
   const infrastructureData = _.get(infrastructure, 'data') as K8sResourceKind;
+  const infrastructurePlatform = getInfrastructurePlatform(infrastructureData);
```

Comment:
@rawagner Please be advised, @jtomasek adds this to `console-shared` selectors in his PR #2558.

https://github.com/openshift/console/pull/2558/files#diff-25304ce4310324e88754e9516a46ac5c

Once #2558 is merged, we should update our existing code to use this shared selector.

This also applies to code below.

Reply:
ok, I will update the existing code then

Reply:
Hmm, I'm wondering if there are any duplicate code detectors for TypeScript / JavaScript, which can calculate code similarity (e.g. percentage) between files.